frontend/public/components/utils/dropdown.jsx

```diff
@@ -411,7 +411,7 @@ ActionsMenu.propTypes = {
 };
 
 const containerLabel = (container) =>
-  <ResourceName name={container.name} kind="Container" />;
+  <ResourceName name={container ? container.name : ''} kind="Container" />;
```

Comment:
Maybe return `null` from render if there are no container statuses yet? Then you don't have the empty dropdown.